components/button/button.tsx

```diff
@@ -141,7 +173,7 @@ const Button = React.forwardRef<HTMLButtonElement, React.PropsWithChildren<Butto
         disabled={disabled}
         onClick={clickHandler}
         {...props}>
-        {loading && <ButtonLoading color={color} />}
+        {loading && <ButtonLoading color={colors.color} />}
         {childrenWithIcon}
```

Comment:
Is it possible or is it the correct way to just use `{!loading && childrenWithIcon}` rather than pass loading into `getButtonChildrenWithIcon`.

Reply:
Let's keep it this way to let users control this.